src/language/code_generator.cmake

```diff
@@ -207,14 +215,6 @@ set(VISITORS_GENERATED_SOURCES
     ${PROJECT_BINARY_DIR}/src/visitors/visitor.hpp
 )
 
-set(PYBIND_GENERATED_SOURCES
-    ${PROJECT_BINARY_DIR}/src/pybind/pyast.cpp
-    ${PROJECT_BINARY_DIR}/src/pybind/pyast.hpp
-    ${PROJECT_BINARY_DIR}/src/pybind/pysymtab.cpp
-    ${PROJECT_BINARY_DIR}/src/pybind/pyvisitor.cpp
-    ${PROJECT_BINARY_DIR}/src/pybind/pyvisitor.hpp
-)
```

Comment:
by the way, there is something wrong with cmake-format that constantly reorder this, have seen something with @cattabiani  as well. Something with the cmake-format version?

Reply:
and same happened in my PR.
Also "there is something wrong with cmake-format" is a tautology...

Reply:
I have noted this issue as well before. I create an issue for this https://github.com/BlueBrain/nmodl/issues/350